prowler/lib/check/checks_loader.py

```diff
@@ -110,15 +112,20 @@ def update_checks_to_execute_with_aliases(
 ) -> set:
     """update_checks_to_execute_with_aliases returns the checks_to_execute updated using the check aliases."""
     # Verify if any input check is an alias of another check
-    for input_check in checks_to_execute:
-        if (
-            input_check in check_aliases
-            and check_aliases[input_check] not in checks_to_execute
-        ):
-            # Remove input check name and add the real one
-            checks_to_execute.remove(input_check)
-            checks_to_execute.add(check_aliases[input_check])
-            print(
-                f"\nUsing alias {Fore.YELLOW}{input_check}{Style.RESET_ALL} for check {Fore.YELLOW}{check_aliases[input_check]}{Style.RESET_ALL}...\n"
-            )
-    return checks_to_execute
+    try:
+        new_checks_to_execute = checks_to_execute.copy()
```

Comment:
Why do you need this copy? If you use a `set` you are not going to have duplicates in that object.

Reply:
Because I only want to modify the copied set, not the original one since it is being iterated.

Reply:
If the original set is modified while iterating through it, it will throw an error.

Reply:
Why would that happen? We were removing checks from the `checks_to_execute`.